groovy/prePostInit/ModifyRecipeMaps.groovy

```diff
@@ -29,6 +29,22 @@ GTRecipeHandler.removeAllRecipes(GTFORecipeMaps.GREENHOUSE_RECIPES);
 GTRecipeHandler.removeAllRecipes(RecipeMaps.VACUUM_RECIPES);
 GTRecipeHandler.removeAllRecipes(RecipeMaps.ELECTROLYZER_RECIPES);
 
+//Add mixer recipes to blender
```

Comment:
why are these lines commented out and added to these files?

Reply:
Not particularly sure, it must be something from some of the previous commits? I'll just remove it now.